test/TEST-41-EXECSTOPPOST/testsuite.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+set -ex
+set -o pipefail
+
+systemd-analyze log-level debug
+systemd-analyze log-target console
```

Comment:
Is this necessary? Based on https://github.com/systemd/systemd/pull/13719#issuecomment-539292650, this make make the test more flaky.

Reply:
No idea. I just copied that from an existing test :-)

Reply:
Well, at least the `set -o pipefail` is not needed, because there are no pipes used in the test. I've dropped that.

Reply:
OK, so please delete the set-log-target part.